blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
 
-        if (plugin == null) {
-            return null;
-        }
 
         try {
-            ResourceBundle resourceBundle = ResourceBundle.getBundle(bundleParams.bundleName, locale, plugin.classLoader);
+            ResourceBundle resourceBundle;
+            if (plugin == null) {
+                Jenkins jenkins = Jenkins.getInstance();
+                if (jenkins!=null && jenkins.pluginManager!=null && jenkins.pluginManager.uberClassLoader!=null) {
```

Comment:
...like in the real world ;) https://www.nytimes.com/2016/12/21/technology/san-francisco-california-uber-driverless-car-.html?_r=0